ray-on-gke/tpu/kuberay-tpu-webhook/main.go

```diff
@@ -590,20 +626,28 @@ func (t *TPUWebhookServer) mutatePod(admissionReview *admissionv1.AdmissionRevie
 	chipsPerHost := getNumTPUChipsRequested(containers...)
 	numOfHosts, _ := getNumTPUHostsFromTopology(clusterName, groupName, namespace, topology, chipsPerHost) // ignore error here because topology may not be set yet
 
-	podsInGroup, err := t.podLister.Pods(namespace).List(labels.SelectorFromSet(labels.Set{"ray.io/group": groupName}))
-	if err != nil {
-		return nil, err
+	// Wait for PodInformer cache to update from previous requests or timeout
+	waitTimeout(&t.wg, time.Second*1)
```

Comment:
Is it expected to run waitTimeout twice, here and the if condition right below?

Reply:
Oh only the one in the if condition was intended, I thought I removed the above line in the commit. [ebe0f33](https://github.com/GoogleCloudPlatform/ai-on-gke/pull/859/commits/ebe0f3304d2c5ce4fd4d662c7a52ed049000ab32)